src/pushover_complete/pushover_api.py

```diff
@@ -4,6 +4,7 @@
     from urlparse import urljoin
 
 import requests
+import os
```

Comment:
I tend to keep imports in order of stdlib, then packages, then internal imports. Could you change to
```python
import os
try: 
    ...
except:
    ...

import requests
```

Reply:
Disagree. It is standard to have the try/except stuff at the end of imports

Reply:
defering to the owner for now